### ISSUE TITLE
Azure Static Websites

### ISSUE THREAD

**Issue by jtheisen** [n3wt0n/AzureWebAppSSLManager#47]
Yet another way of deploying websites on Azure is [Azure Static Web Apps](https://learn.microsoft.com/en-us/azure/static-web-apps/). Again there are custom domains and certificates to be managed and bound.

The service isn't really part of App Services, but some things are related. Especially it can also use App Service Certificates.

I really like ASWs, but I'd also like to have my certs renewed without buying the expensive ones from Azure.

I'm wondering what I would have to do to make this library refresh the bindings in ASWs - the other two steps (cert and dns updates) should be the same, right?

I had some brief look at the source code and the Azure Docs for ASW, but so far I can't find C# bindings for the ASW REST API and I also can't find cert binding functions in the REST API itself.

I'm writing this both as a feature request and maybe someone already has a bit of info what needs to be done here.

**Comment by n3wt0n** [comment]
Good point, I haven't looked into that yet myself... but it could be interesting
